impl/src/main/java/org/jboss/weld/logging/BootstrapLogger.java

```diff
@@ -276,4 +276,8 @@
     @Message(id = 166, value = "AfterTypeDiscovery.{3} modified by {0} {2} {1}", format = Format.MESSAGE_FORMAT)
     void typeModifiedInAfterTypeDiscovery(Object extensionName, Object type, Object operation , Object types);
 
+    @LogMessage(level = Level.WARN)
+    @Message(id = 167, value = "Class {0} cannot be resolved to a bean because it does not declare appropriate constructor!", format = Format.MESSAGE_FORMAT)
+    void annotatedTypeNotResolvedToBean(String clazzName);
```

Comment:
The method name and the message should be more specific. Maybe something like _"Class {0} is annotated with {1} but it does not declare an appropriate constructor therefore is not registered as a bean"_...